app/src/main/java/ca/ualberta/CMPUT3012019T02/alexandria/model/BookList.java

```diff
@@ -0,0 +1,66 @@
+package ca.ualberta.CMPUT3012019T02.alexandria.model;
+
+import android.net.Uri;
+
+import ca.ualberta.CMPUT3012019T02.alexandria.R;
+
+public class BookList {
+    private int cover;
```

Comment:
`private int cover;` --> `private Bitmap cover;`
Images will be supplied using `Bitmap`.